### PR TITLE
Delegation02.md updated. Specification fix for issue #140 in the engine

### PR DESCRIPTION
**Pull request by sandoche2k**

### CODE REVIEW

File: docs/specifications/tests/Delegation-TP/delegation02.md [hunk 1]
```diff
@@ -23,8 +23,11 @@ The domain name to be tested.
 1. Obtain the IP addresss of the name servers from the parent using  [Method
 4](../Methods.md#method-4-obtain-glue-address-records-from-parent) and the child using
    [Method 5](../Methods.md#method-5-obtain-the-name-server-address-records-from-child).
-2. If any of the IP addresses resolved in step 1 is not unique, then this
    test case fails.
+2. If the number of IP addresses obtained is less than two, this test case fails
+   with an error.
+3. If any of the IP addresses resolved in step 1 is not unique, then this
+   test case fails with a warning.
```
Comment (matsduf) [inline docs/specifications/tests/Delegation-TP/delegation02.md]: I do not think that the steps in the new text supports the objective of the Test Case: "If the domain have several different name server names used, they can all be using the same IP address. (...) This test case checks that the name servers used does not resolve to reuse the same IP addresses."

I think the steps should collect nameserver names and IP addresses and then check if any address is referred to more than once.

@sandoche2k, do you agree?

**Comment by sandoche2k** [comment]
@matsduf please merge

File: docs/specifications/tests/Delegation-TP/delegation02.md (hunk 1 above)
Comment (sandoche2k) [reply docs/specifications/tests/Delegation-TP/delegation02.md]: @sandoche2K - To review

**Comment by sandoche2k** [comment]
@matsduf this pull request can be closed on reviewing https://github.com/dotse/zonemaster/pull/507

**Comment by matsduf** [comment]
Closed in request.

**Pull request closed without merging**